firebase-sessions/src/test/kotlin/com/google/firebase/sessions/SessionInitiatorTest.kt

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.sessions
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+class SessionInitiatorTest {
+  class FakeTime {
+    var currentTimeMs = 0L
+      private set
+
+    fun addTimeMs(intervalMs: Long) {
+      currentTimeMs += intervalMs
+    }
+  }
```

Comment:
Do you see a need for reusing this? May be we can create set of Mocks/Fakes that could be reused? Not for now - but for later.

Reply:
Yeah if this is used somewhere else, I'll pull it out into a top level class and maybe make a package like `testing`.

Reply:
Sounds good, thanks.